doubly_linked_list/doubly_linked_list.py

```diff
@@ -3,10 +3,16 @@
 as well as its next node in the List.
```

Comment:
I know you're still working on this, so I'll hold off leaving any comments for now.

Reply:
NICE 🎉  Awesome job!